mautic/mautic
# Fix miscellaneous tag related issue

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Yes
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | Yes 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Fix following tag related issues :-
1. Removed description box from tag add/edit/list/detail page
2. Save and Close button doesn't work

> 1. Go to tags
> 2. Add New
> 3. Enter value 1234 save and close
> 4. Add new
> 5. Enter value 1234 save and close



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
4. Tag add/edit/detail should work properly
